src/Sulu/Bundle/CustomUrlBundle/Controller/CustomUrlController.php

```diff
@@ -109,16 +109,13 @@ public function getAction($webspace, $id, Request $request)
     /**
      * Create a new custom-url object.
      *
-     * @param string $webspace
-     *
      * @return Response
      */
-    public function postAction($webspace, Request $request)
+    public function postAction(string $webspace, Request $request)
```

Comment:
we cannot change the method signature because it would be a bc break, so we need to revert this 😕 
```suggestion
    public function postAction($webspace, Request $request)
```

Reply:
The function what is called here only has two parameters

Reply:
yes, changing the call below is fine 🙂  but we cannot add the `string` typehint for the `$webspace`